examples/retinanet/configs/default.py

```diff
@@ -0,0 +1,50 @@
+from jax.numpy import float32
+
+
+class ConfigDict(dict):
```

Comment:
ConfigDict is now open sourced:
https://pypi.org/project/ml-collections/

Reply:
Should be fixed now.

Reply:
Thanks!